config_defaults/subtests/docker_cli/systemd.ini

```diff
@@ -0,0 +1,20 @@
+[docker_cli/systemd]
+#: Systemd unit file ``sysd_unit_file``
+sysd_unit_file = p4321.service
+#: Systemd unit file destination ``sysd_unitf_dest``
+sysd_unitf_dest = /etc/systemd/system/p4321.service
+
+#: increase ``docker_timeout`` to 180 as building image can take longer
+docker_timeout = 300
+
+#: build image ``name``
+name=p4321
+
+#: image building args ``build_options_csv``
+build_options_csv = --force-rm,-t,%(name)s
+
+#: local ip address ``tcp_ip``
+tcp_ip = 127.0.0.1
+
+#: port to use ``tcp_port``
```

Comment:
@afomm I would assume those Travis warnings come from `./run_checkdocs.py` but I've been unable to reproduce them. BTW what I'm referring to is the backquote-backquote-varname-backquote-backquote strings. Try removing those, writing full param descriptions, and run `./run_checkdocs.py` on your system to confirm they're ok.

@cevich, nice suggestion. Indeed there's no need to make 127.0.0.1 and 4321 configurable.

Also, the comment on docker_timeout is misleading.